core/modules/widgets/edit-text.js

```diff
@@ -230,12 +243,11 @@ EditTextWidget.prototype.fixHeight = function() {
 		var newHeight = Math.max(domNode.scrollHeight + domNode.offsetHeight - domNode.clientHeight,minHeight);
 		// Only try to change the height if it has changed
 		if(newHeight !== domNode.offsetHeight) {
-			domNode.style.height =  newHeight + "px";
+			domNode.style.height = newHeight + "px";
 			// Make sure that the dimensions of the textarea are recalculated
 			$tw.utils.forceLayout(domNode);
-			// Check that the scroll position is still visible before trying to scroll back to it
-			scrollTop = Math.min(scrollTop,self.document.body.scrollHeight - window.innerHeight);
```

Comment:
Hi @felixhayashi my apologies. I'm going to focus on the backlog of pull requests for a few days, and wanted to start with this one!

Anyhow, your changes appear to have lost the logic here that checks that the scroll position is still visible. Is that intentional?